mobify/mobify-code-style
# ✨ Add eslint rule to enforce comment whitespacing

Let's enforce comment whitespacing. We already enforce this rule in ES6. As it is backwards-compatible with pre-v1 versions of eslint, let's include it there too.
## Changes
- Add `spaced-comment` eslint rule to `/javascript/.eslintrc` and `/es5/mobify-es5.yml` for enforcing (rule is already included in ES6 ruleset)
- Add comments to `/javascript/README` and `/es5/README` regarding rule enforcement
## How To Test
- Checkout code
- `npm link`
- In a project using ES5 linting, run `npm link mobify-code-style`
- Make a comment without adding whitespace to the starting comment marker (`//Bad comment`)
- `grunt lint`
- Ensure comment whitespacing is enforced
## Applicable Research Resources
- http://eslint.org/docs/rules/spaced-comment
## TODOs:
